Description: Do not run toplevel tests
 Unit tests under the root folder of the source package are using the
 namespace, which breaks under Python 3. This patch avoids doing discovery
 of tests outside of oslo_middleware, which is the new place where patches
 are done (others are useless or obsolete).
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-04-08

--- python-oslo.middleware-1.0.0.orig/.testr.conf
+++ python-oslo.middleware-1.0.0/.testr.conf
@@ -2,6 +2,6 @@
 test_command=OS_STDOUT_CAPTURE=${OS_STDOUT_CAPTURE:-1} \
              OS_STDERR_CAPTURE=${OS_STDERR_CAPTURE:-1} \
              OS_TEST_TIMEOUT=${OS_TEST_TIMEOUT:-60} \
-             ${PYTHON:-python} -m subunit.run discover -t ./ . $LISTOPT $IDOPTION
+             ${PYTHON:-python} -m subunit.run discover -t ./ ./oslo_middleware $LISTOPT $IDOPTION
 test_id_option=--load-list $IDFILE
 test_list_option=--list
